cw_nano/lib/pending_nano_transaction.dart

```diff
@@ -0,0 +1,42 @@
+import 'package:cw_core/pending_transaction.dart';
+import 'package:cw_nano/nano_client.dart';
+import 'package:cw_nano/nano_util.dart';
+
+class PendingNanoTransaction with PendingTransaction {
+  PendingNanoTransaction({
+    required this.nanoClient,
+    required this.amount,
+    required this.fee,
+    required this.id,
+    required this.blocks,
+  });
+
+  final NanoClient nanoClient;
+  final BigInt amount;
+  final int fee;
+  final String id;
+  final List<Map<String, String>> blocks;
+  String hex = "unused";
+
+  @override
+  String get amountFormatted {
+    final String amt = NanoUtil.getRawAsUsableString(amount.toString(), NanoUtil.rawPerNano);
+    return amt;
+  }
+
+  String get accurateAmountFormatted {
+    final String amt = NanoUtil.getRawAsUsableString(amount.toString(), NanoUtil.rawPerNano);
+    final String acc = NanoUtil.getRawAccuracy(amount.toString(), NanoUtil.rawPerNano);
+    return "$acc$amt";
+  }
```

Comment:
this checks whether the displayed amount === RAW amount, and adds a tilde to represent that it isn't exact if necessary
basically, if there's an amount past 6 decimal places, amt = "~"
so if the amount is 100.123456 that will display as 100.123456
but if the amount is 100.1234567 it will return ~100.123456
the amount is NOT rounded